app/src/main/java/me/phh/treble/app/LenovoSettings.kt

```diff
@@ -0,0 +1,17 @@
+package me.phh.treble.app
+
+import android.os.Bundle
+import android.preference.PreferenceFragment
+
+object LenovoSettings {
+    val dt2w = "lenovo_double_tap_to_wake"
+
+    fun enabled(): Boolean = Tools.vendorFp.contains("Lenovo")
```

Comment:
Depending on your answer for dtPanel, I might want a `&& File(dtPanel).exists()`